classes/class-ed-lti.php

```diff
@@ -384,6 +384,9 @@ private function render_student_blogs_list_view( $course_id, $resource_link_id )
 		);
         // phpcs:enable
 
+        // Cache the response for 30 minutes
+        header('Cache-Control: private, max-age: 1800');
```

Comment:
Looks good, but on github the formatting is off. Looks like it needs tabbed in, so it is inline with the rest of the code

Reply:
Hi @richeduni - I've fixed the formatting now. Sorry, I had aligned it with the comment above. I've also fixed the that comment's alignment. If you could take a second look that would be greatly appreciated.

All the best,
Andrew